src/query.rs

```diff
@@ -21,8 +21,17 @@ impl RequiredSemverUpdate {
     }
 }
 
+impl From<RequiredSemverUpdate> for ReleaseType {
+    fn from(value: RequiredSemverUpdate) -> Self {
+        match value {
+            RequiredSemverUpdate::Major => Self::Major,
+            RequiredSemverUpdate::Minor => Self::Minor,
+        }
+    }
+}
+
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub(crate) enum ActualSemverUpdate {
+pub enum ActualSemverUpdate {
```

Comment:
Would you mind adding a doc comment here, since this type is new to the public API?